### PR TITLE
Doc: Clear type errors and warnings in the rescript playground where posible

### LINKED ISSUE #600: Type errors, Warnings shown in the ReScript Playground for snippets in the Exceptions documentation page
Actual

- Go to https://rescript-lang.org/docs/manual/latest/exception
- Run the first code example in the ReScript playground.
- See the type error in Javascript tab
- See `unused variable items` warning in the Problems tab

Expected

- Compiler should finish without errors/warnings

Beginners are stumped as they cannot try the examples listed and instead spend time having to fix the code.

Remedy 

- Code examples must contain all required/dependent Types/Methods/Externals  before running in the Rescript Playground

### PR DESCRIPTION
**Pull request by kamweti**
fixes #600

### CODE REVIEW

File: pages/docs/manual/latest/exception.mdx [hunk 1]
```diff
@@ -62,20 +65,32 @@ You can directly match on exceptions _while_ getting another return value from a
 <CodeTab labels={["ReScript", "JS Output"]}>
 
 ```res
```
Comment (cristianoc) [inline pages/docs/manual/latest/exception.mdx:67]: should this be `prelude` ?

Comment (cristianoc) [reply pages/docs/manual/latest/exception.mdx:67]: @kamweti reopening the question

Comment (kamweti) [reply pages/docs/manual/latest/exception.mdx:67]: @cristianoc sorry i thought i posted the comment earlier

Comment (cristianoc) [reply pages/docs/manual/latest/exception.mdx:67]: `prelude` is exactly what makes sure future changes don't introduce compilation errors on the code
That's why I suggested to add it

Comment (kamweti) [reply pages/docs/manual/latest/exception.mdx:67]: ok sure, i'm gonna add it

**Review by cristianoc — APPROVED** [review]
Thanks a lot for the fix!

**Review by kamweti — COMMENTED** [review]
adding `prelude` in one more place

**Comment by kamweti** [comment]
@cristianoc 👍🏿

**Pull request merged**